gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/OrganizationUserService.java

```diff
@@ -38,6 +40,8 @@ public interface OrganizationUserService extends CommonUserService {
 
     Single<User> updateLogoutDate(ReferenceType referenceType, String referenceId, String id);
 
+    Single<List<AccountAccessToken>> findAccountAccessTokens(String organizationId, String userId);
```

Comment:
question: why not use `Flowable<AccountAccessToken>?`

Reply:
It can be Flowable. The update is pushed.